driver-core/src/main/java/com/datastax/driver/core/TableMetadata.java

```diff
@@ -416,6 +423,9 @@ public String exportAsString() {
 
```

Comment:
Maybe we should add a `TableMetadata.isVirtual()`, even though it will just be a trivial shortcut to `getKeyspace().isVirtual()`.